features/step_definitions/work_download_steps.rb

```diff
@@ -1,12 +1,12 @@
 Then /^I should see the inspiring parent work link$/ do
   parent = Work.find_by(title: "Worldbuilding")
-  inspired_link = "<a href=\"#{work_url(parent)}\">#{parent.title}</a>"
+  inspired_link = "<a href=\"#{UrlFormatter.new(work_url(parent)).with_https}\">#{parent.title}</a>"
```

Comment:
we don't really use urlformatter for things like this, it's not quite what it's made for. I think you could hardcode this url with `www.example.com`, we also do that in some other tests